### ISSUE TITLE
cis/trans info lost in bond breaking reaction

### ISSUE THREAD

**Issue by mc-robinson** [rdkit/rdkit#2885]
rdkit 2019.03.2

**Description:**

Hi, I am trying to perform the following bond breaking transformation 
rxn = AllChem.ReactionFromSmarts("[CX4;H3,H2,H1:1][c,C:2]>>[C:1][Br].[Li][c,C:2]")

on 'C=C(CCO)C(C)/C(C)=C/Cl'
![image](https://user-images.githubusercontent.com/10458975/72513726-44961d00-3802-11ea-95b8-2d592f64a5e2.png)

Obviously, this will break carbon-carbon bonds at many places. But one product should be 
"BrC(C)C(CCO)=C.C/C([Li])=C\\Cl"
![image](https://user-images.githubusercontent.com/10458975/72513850-80c97d80-3802-11ea-84a1-8865dcf0bb2f.png)

This correct products, ['C=C(CCO)C(C)Br', '[Li]/C(C)=C/Cl'] are produced by rdkit, however, stereochemistry information seems to be lost on the alkene species.
rdkit result: ['C=C(CCO)C(C)Br', '[Li]C(C)=CCl'].

My understanding was that stereochemistry should be maintained, since I do not specify a change in the reaction.

I have tried broadening the Smarts string with dummy atoms to perhaps capture stereochemistry information in adjacent atoms, but that does not seem to fix things. Is it that the broken bond was the bond that was defining the stereochemistry? Any help is greatly appreciated.

Code to reproduce:
```
rxn = AllChem.ReactionFromSmarts("[CX4:1][c,C:2]>>[C:1][Br].[Li][*:2]")
r = Chem.MolFromSmiles('C=C(CCO)C(C)/C(C)=C/Cl')
print(Chem.MolToSmiles(rxn.RunReactants((r,))[5][1]))
```

**Comment by d-b-w** [comment]
MC -
I believe that this is fixed in the 2019.09 release (specifically, this
enhancement: https://github.com/rdkit/rdkit/pull/2553). You might try
updating RDKit and double checking.

I see the following in a current RDKit:

from rdkit import Chem
from rdkit.Chem import AllChem
rxn = AllChem.ReactionFromSmarts("[CX4:1][c,C:2]>>[C:1][Br].[Li][*:2]")
reactant = Chem.MolFromSmiles('C=C(CCO)C(C)/C(C)=C/Cl')
print(Chem.MolToSmiles(rxn.RunReactant(reactant, 0)[5][1]))
"[Li]/C(C)=C/Cl"



*dan nealschneider* | lead developer
[image: Schrodinger Logo] <https://www.schrodinger.com/>

**Comment by mc-robinson** [comment]
Thank you! After getting all my conda issues straightened out, and updating, this seemed to fix it.
